completed-plant.css

```diff
@@ -4,7 +4,9 @@ body {
 .bg {
   margin: auto;
   position: absolute;
-  background-color: #CFAC64;
```

Comment:
i dont know why this is showing like this in the pull request. it's not in the code that way, it's definitely commented out. *shruggies*